example/bench/bench.cpp

```diff
@@ -0,0 +1,78 @@
+/*
+ * Miniapp that uses the artificial benchmark cell type to test
+ * the simulator infrastructure.
+ */
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+
+#include <json/json.hpp>
+
+#include <common_types.hpp>
+#include <communication/distributed_context.hpp>
+#include <hardware/node_info.hpp>
+#include <load_balance.hpp>
+#include <profiling/meter_manager.hpp>
+#include <simulation.hpp>
+#include <util/ioutil.hpp>
+
+#include "recipe.hpp"
+
+using namespace arb;
+
+int main(int argc, char** argv) {
+    try {
+        distributed_context context;
+        #ifdef ARB_HAVE_MPI
```

Comment:
Do we have a standard for #ifdef alignment? I tend to like them flush left, since they are pre-processing, rather than part of the code logic itself.